content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,176 @@ ifdef::backend-html5[]
 :toc: left
 endif::[]
 
-= Administering Jenkins on Kubernetes
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* How to upgrade the version of Kubernetes in your cluster.
+* Upgrade Kubernetes cluster maintained by the Jenkins Infrastructure project.
+* Upgrade Jenkins Inside a Docker Container.
+* Kubernetes Concepts.
+
+== Upgrading from one version of Kubernetes to another:
+
+Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+A Kubernetes cluster adds a new automation layer to Jenkins.
+Kubernetes makes sure that resources are used effectively and that your servers and underlying infrastructure are not overloaded.
+Kubernetes' ability to orchestrate container deployment ensures that Jenkins always has the right amount of resources available.
+
+=== How to upgrade the version of Kubernetes in your cluster:
+
+Upgrading your Kubernetes cluster is one of the tasks you must take on as an operator of Jenkins X. 
+Kubernetes comes out with link:https://kubernetes.io/releases[new releases] regularly, and will only maintain releases for the three latest versions.
```

Comment:
Phrasal verbs like "comes out", "get over", and "go under" can be more difficult for non-native speakers because they require two words to understand the verb.  Better to use a simple verb when we can.
```suggestion
Kubernetes delivers link:https://kubernetes.io/releases[new releases] regularly, and will only maintain releases for the three latest versions.
```